pkg/cli/root.go

```diff
@@ -69,7 +69,18 @@ var rootCmd = func() cli.Command {
 				}
 			},
 			"token": func() cli.Command {
-				return &TokenCommand{}
+				return &cli.RootCommand{
+					Name:        "token",
+					Description: "Perform token operations",
+					Commands: map[string]cli.CommandFactory{
+						"create": func() cli.Command {
+							return &TokenCommand{}
```

Comment:
nit: Can we now rename this struct to `TokenCreateCommand`?

Reply:
Yes, I will rename in a separate followed-up PR so it is easier to see all the other relevant changes.